Automattic/woocommerce-payments
# Fix generate receipt preview in FF

Fixes #4070 

#### Changes proposed in this Pull Request


This PR fixes an issue with the preview of the printed receipt in the card readers page. It fails to render in the FF browser due to a [documented bug in React](https://github.com/facebook/react/issues/22847). To fix the issue I use an approach suggested as a workaround in the same issue.







#### Testing instructions





* Using Firefox browser, navigate to WP Admin > Payments > Card Readers
* Go to Receipt Details tab
* Click Preview a printed receipt link
* Check that the preview is generated correctly

I tested it in Firefox, Chrome and Safari and works as expected.

-------------------


**Post merge**
